### ISSUE TITLE
Update to the latest TZ Shapefiles

### ISSUE THREAD

**Issue by mvrhov** [drtimcooper/LatLongToTimezone#9]
Unfortunately, the timezones are changed in various countries a couple a times a year. So the TZ database gets updated. It would be nice to update the data to latest TZ shapefiles at least twice a year.

**Comment by sankarsana** [comment]
I join the request! Please fix it.

**Comment by stephent** [comment]
It appears that regularly updated shapefiles are available from here: [https://github.com/evansiroky/timezone-boundary-builder](https://github.com/evansiroky/timezone-boundary-builder). Could they function as an updated source for this project?

**Comment by darpandhamija** [comment]
I have a fix available for it. I've picked the latest shapefiles and updated this library with the latest timezone. @drtimcooper How can I contribute ?

**Comment by drtimcooper** [comment]
@darpandhamija Have you executed the code to create the target java/csharp/objectivec code?  Does the new version of the files work?  You should test it with at least one of the supported languages.  If it works then: are you familiar with git?  Can you create a git pull request?  I'll merge it if you're confident that it works.

**Comment by darpandhamija** [comment]
Yes @drtimcooper, I had to update the convertingShapeFilesToJson, 
- I've upgraded to latest ShapeFile lib version. 
- I had to update toJson.c for creating polygons from new shapefiles
- update paths on TimeZoneMapperConverter.java to more generic ones to finally generate .java and .swift files. 
- I've added some more test cases for the file and compared the result with other libraries and Google Timezone API response and it has improved results as compared to before. Something that'll resolve another issue : https://github.com/drtimcooper/LatLongToTimezone/issues/19

I am familiar with git and create PullRequest for you to merge.

**Comment by drtimcooper** [comment]
Hi @darpandhamija  : Thanks for your hard work!   But I can't see your Pull Request.  I can only see one PR and it's not yours.  Can you try again?
